doc/planning/RequestBehaviourChangeService.md

```
@@ -0,0 +1,24 @@
+This service is hosted in the node RequestBehaviourChangeService.
```

Markdownlint (0.35.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)